common/connect.go

```diff
@@ -36,22 +38,63 @@ func (tb *tunnelBroadcaster) Accept(e event.E) {
 	}
 }
 
-func ConfigureCertPool(customCertificateFilePath string) {
-	// Get the SystemCertPool, continue with an empty pool on error
+// ConfigureCertPool trusts given certificates
+// CAFilePath is a path to file which contains CA certificates.
+//   Usually it contains Che server self-signed certificate.
```

Comment:
Please consider propagating it to a more visible place, like flag description.

Reply:
ok